src/input/base.ts

```diff
@@ -50,12 +53,9 @@ export abstract class LocalInputSource extends InputSource {
   public fileObject!: Buffer | string;
 
   /**
-   * @param {String} inputType - the type of input used in file ("base64", "path", "dummy").
-   *                             NB: dummy is only used for tests purposes
-   * @param {Boolean} cutPages
-   * NB: Because of async calls, init() should be called after creating the object
+   * @param {InputConstructor} constructor Constructor parameters.
    */
-  constructor({ inputType }: InputProps) {
+  protected constructor({ inputType }: InputConstructor) {
```

Comment:
Is this really needed on abstract class?

Reply:
no, not required, just my IDE was nagging me about it

Reply:
Yeah fair enough